google/gax/api_callable.py

```diff
@@ -30,107 +30,17 @@
 """Provides function wrappers that implement page streaming and retrying."""
 
 from __future__ import absolute_import, division
-import random
-import time
 
 from future import utils
 
-from . import (BackoffSettings, BundleOptions, bundling, _CallSettings, config,
-               errors, PageIterator, ResourceIterator, RetryOptions)
-from .errors import RetryError
+from google.gax import (BackoffSettings, BundleOptions, bundling, _CallSettings,
+                        config, errors, PageIterator, ResourceIterator,
+                        RetryOptions)
+from google.gax.retry import add_timeout_arg, retryable
```

Comment:
Style: `from google.gax import retry`

(Note: [Google style](https://google.github.io/styleguide/pyguide.html#Imports_formatting) is only to import modules and packages; in this project, we additionally allow classes.)